cmake/ycm-0.3.1/find-modules/FindZFP.cmake

```diff
@@ -27,27 +27,28 @@
 include(FindPackageHandleStandardArgs)
 include(SelectLibraryConfigurations)
 
-find_path(ZFP_INCLUDE_DIR
+find_path(ZFP_INCLUDE_DIRS
```

Comment:
Can you please better explain why is it wrong?
Before it was:
```
find_path(ZFP_INCLUDE_DIR
          NAMES zfp.h
          PATHS $ENV{ZFP_ROOT}/inc
          DOC "ZFP include directory")
...
set(ZFP_INCLUDE_DIRS ${ZFP_INCLUDE_DIR})
```
The last line looks like an addictional line with no actual effect to me.

Now it is:
```
find_path(ZFP_INCLUDE_DIRS
          NAMES zfp.h
          PATHS $ENV{ZFP_ROOT}/inc
          DOC "ZFP include directory")
```
The result should be the same imho. What am I missing?